src/functionaltest/java/com/ericsson/ei/frontend/SubscriptionHandlingFunctionality.java

```diff
@@ -204,12 +204,24 @@ public void testSubscription() throws Exception {
         String requirementFieldID = "requirementID";
 
         subscriptionPage.selectDropdown(selectAuthID, authValue);
-
         subscriptionPage.addFieldValue(userNameID, userName);
         subscriptionPage.addFieldValue(tokenID, token);
         String kvID = "kvID";
-        subscriptionPage.clickKVbtn(kvID);
+        subscriptionPage.clickKVbtn(kvID);        
```

Comment:
Trailing white spaces here.
It is possible to automatically remove trailing white spaces in eclipse when you save.